src/frontends/tensorflow_common/include/helper_ops/string_constant.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+/// Pseudo-entity for storing strings
+class StringConstant : public InternalOperation {
+public:
+    OPENVINO_OP("StringConstant", "ov::frontend::tensorflow::util", InternalOperation);
+
+    StringConstant(ov::Any data, const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1),
+          m_data(data),
+          m_str() {
+        validate_and_infer_types();
+    }
+
+    StringConstant(std::string& str, const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1),
+          m_data(),
+          m_str(str) {
+        validate_and_infer_types();
+    }
+
+    StringConstant(const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1) {
+        validate_and_infer_types();
+    }
+
+    void validate_and_infer_types() override {
+        set_output_type(0, ov::element::undefined, ov::PartialShape::dynamic());
+    }
+
+    ov::Any get_data() {
+        return m_data;
+    }
+
+    std::string& get_string() {
+        return m_str;
+    }
+
+private:
+    ov::Any m_data;
+    std::string m_str;
```

Comment:
why not `m_str` is sufficient?

Reply:
I see two types of possible implementations - when it returns vector of strings. I don't want to join them and split later.

Biggest question - why do we need ov::Any here - but it was you suggestion to keep vector in ov::Any.

Solution with everytime check "ov::Any.is<std::string>" or "ov::Any.is<std::vector<std::string>>" looks complicated and ugly.